funnel/assets/js/util.js

```diff
@@ -166,13 +173,13 @@ export const Utils = {
       let eventDay = Date.parse(eventDate);
       // Find the difference between event and today's date in UTC
       let counting = Math.round((eventDay - today) / singleDay);
-      let dayText = ['Today', 'Tomorrow', 'Day after'];
+      // Defined these strings in project_countdown macro in calendar_snippet.js.jinja2
+      let dayText = ['Today', 'Tomorrow', 'Day after', 'In %d days'];
       // Show number of days on the widget only if it is less than 32 days
       if (counting >= 0 && counting < 3) {
-        monthElem.text(dayText[counting]);
+        monthElem.text(window.gettext(dayText[counting]));
       } else if (counting > 2 && counting < 32) {
-        let daysRemainingTxt = `In ${counting} days`;
-        monthElem.text(daysRemainingTxt);
+        monthElem.text(window.gettext(dayText[3], counting));
```

Comment:
Looks like this will mean the source texts will not be detected by `make babel` as there is no gettext wrapper on them. Are these strings hardcoded into the `pot` file, or are there copies elsewhere?

Reply:
Found it in the calendar snippet template.